addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,779 @@
+        SET supplier_rank = rel.supplier_rank
+        FROM (SELECT am.partner_id, count(am.id) as supplier_rank
+            FROM account_move am
+            JOIN res_partner rp ON am.partner_id = rp.id
+            WHERE am.state != 'draft' AND left(am.type,3) = 'in_' AND
+                am.partner_id IS NOT NULL
+            GROUP BY am.partner_id) rel
+        WHERE rel.partner_id = rp.id""",
+    )
+
+
+def try_to_fill_account_account_tag_country_id(env):
+    # NOTE: the filling is not exhaustive
+    # from account.account
+    openupgrade.logged_query(
+        env.cr, """
+        WITH select_tags AS (
+            SELECT aat.id
+            FROM account_account_tag aat
+            JOIN account_account_account_tag aaat ON (
+                aat.applicability = 'accounts' AND aat.country_id IS NULL
+                AND aaat.account_account_tag_id = aat.id)
+            JOIN account_account aa ON aaat.account_account_id = aa.id
+            JOIN res_company rc ON aa.company_id = rc.id
+            JOIN res_partner rp ON (
+                rc.partner_id = rp.id AND rp.country_id IS NOT NULL)
+            GROUP BY aat.id
+            HAVING count(DISTINCT rp.country_id) = 1
+        )
+        UPDATE account_account_tag aat
+        SET country_id = rp.country_id
+        FROM account_account_account_tag aaat
+        JOIN select_tags sel ON sel.id = aaat.account_account_tag_id
+        JOIN account_account aa ON aaat.account_account_id = aa.id
+        JOIN res_company rc ON aa.company_id = rc.id
+        JOIN res_partner rp ON rc.partner_id = rp.id
+        WHERE aat.applicability = 'accounts' AND aat.country_id IS NULL
+            AND aaat.account_account_tag_id = aat.id
+            AND rp.country_id IS NOT NULL""",
+    )
+    # from account.tax
+    openupgrade.logged_query(
+        env.cr, """
+        WITH select_tags AS (
+            SELECT aat.id
+            FROM account_account_tag aat
+            JOIN account_tax_account_tag atat ON (
+                aat.applicability = 'taxes' AND aat.country_id IS NULL
+                AND atat.account_account_tag_id = aat.id)
+            JOIN account_tax t ON atat.account_tax_id = t.id
+            JOIN res_company rc ON t.company_id = rc.id
+            JOIN res_partner rp ON (
+                rc.partner_id = rp.id AND rp.country_id IS NOT NULL)
+            GROUP BY aat.id
+            HAVING count(DISTINCT rp.country_id) = 1
+        )
+        UPDATE account_account_tag aat
+        SET country_id = rp.country_id
+        FROM account_tax_account_tag atat
+        JOIN select_tags sel ON sel.id = atat.account_account_tag_id
+        JOIN account_tax t ON atat.account_tax_id = t.id
+        JOIN res_company rc ON t.company_id = rc.id
+        JOIN res_partner rp ON rc.partner_id = rp.id
+        WHERE aat.applicability = 'taxes' AND aat.country_id IS NULL
+            AND atat.account_account_tag_id = aat.id
+            AND rp.country_id IS NOT NULL""",
```

Comment:
I think we can remove it because we have this filter in `select_tags` table and inner join with this table